de_web_plugin.cpp

```diff
@@ -6170,27 +6193,28 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                     {
                         fpSwitch.inClusters.push_back(ci->id());
                     }
-                    else if (modelId == QLatin1String("lumi.sensor_switch.aq3") || modelId == QLatin1String("lumi.remote.b1acn01") ||
```

Comment:
We have regression here, since v2.13.3-beta at least `lumi.remote.b1acn01` aka Xiaomi WXKG11LM 2018 can't be joined anymore. In the debugger I could trace that `fpSwitch` is never filled likely since the model id check was removed here.

As far as I can tell only this switch is affected.